tutorials/adding_system_plugins.md

```diff
@@ -0,0 +1,132 @@
+\page adding_system_plugins
+
+# Overview
+
+This tutorial explains how to add plugins to your model to provide extra
+capabilities to it.
+
+## Related tutorials
+
+https://gazebosim.org/api/sim/8/createsystemplugins.html
+
+# Adding a system plugin
+
+[This Gazebo tutorial](https://gazebosim.org/api/sim/8/createsystemplugins.html)
+describes what is a system plugin in depth. Intuitively, you can envision a
+system plugin as a piece of code that modifies the behavior of the simulation
+when the general physics engine does not exactly capture your needs.
+In our example, our turtle does not move because it's configured as a static
+model. Let's see what happens if you remove that tag.
+
+Modify your `~/gazebo_maritime/models/my_turtle/model.sdf` and remove the line
+ `<static>true</static>`. Then launch the simulation:
```

Comment:
maybe have a Prerequisite section or add a line in the Related tutorials section mentioning that you need to complete the `Create a maritime vehicle` tutorial first? For users who just happen to bump into this page may not know where the `~/gazebo_maritime/models/my_turtle/model.sdf` comes from

Reply:
Changed in [ef9be7b](https://github.com/gazebosim/gz-sim/pull/2257/commits/ef9be7b25be46878ea6a90219cec288e41257826).